src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -503,8 +520,60 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
                 TimeSpan timerInterval = timer == null ? TimeSpan.MinValue : TimeSpan.FromMilliseconds(timer.Interval);
                 try
                 {
-                    await InvokeAsync(invoker, parameterHelper, timeoutTokenSource, functionCancellationTokenSource,
-                    throwOnTimeout, timerInterval, instance);
+                    // A variable for the function result
+                    FunctionResult functionResult = null;
+
+                    // Create a dictionary that will pass the user properties from the Executing filter to the Executed filter
+                    Dictionary<string, object> properties = new Dictionary<string, object>();
+
+                    // Create the context objects for the filter execution
+                    FunctionExecutingContext executingContext = new FunctionExecutingContext(instance.Id, instance.FunctionDescriptor.FullName, parameterHelper.GetParametersAsDictionary(), properties, logger);
+                    executingContext.JobHost = jobHost;
+                    executingContext.Config = config;
+
+                    FunctionExecutedContext executedContext = new FunctionExecutedContext(instance.Id, instance.FunctionDescriptor.FullName, parameterHelper.GetParametersAsDictionary(), properties, logger, functionResult);
```

Comment:
You're assigning null to functionResult a few lines above, which means you're passing this null into this constructor. That is incorrect - the constructor shouldn't allow null and should throw ArgumentNullException if a null is passed in.